.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Before opening an issue**
+
+Questions should go to the GEE mailing list (https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/gee-oss)
+or Slack Open GEE channel (http://slack.opengee.org/) GitHub issues are for bug reports and suggestions for new features.
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+1. Start Open GEE 5.2.5
+2. Create an 3D imagery resource from "myimage.tif" 
+3. I get this error: "....."
+
+**Expected behavior**
+
+A clear and concise description of what you expected to happen.
+For example (please modify !!!): I expected to be able to create imagery resource from my image (with a link to the image file,
```

Comment:
Again, I don't think the example is necessary

Reply:
Ok